README.md

```diff
@@ -1,13 +1,12 @@
-# GraSPy
+# graspologic
 [![Paper shield](https://img.shields.io/badge/JMLR-Paper-red)](http://www.jmlr.org/papers/volume20/19-490/19-490.pdf)
-[![Downloads shield](https://img.shields.io/pypi/dm/graspy.svg)](https://pypi.org/project/graspy/)
-[![Build Status](https://travis-ci.org/neurodata/graspy.svg?branch=master)](https://travis-ci.org/neurodata/graspy)
+[![Downloads shield](https://img.shields.io/pypi/dm/graspologic.svg)](https://pypi.org/project/graspologic/)
```

Comment:
Is there anything you guys want to do around the downloads of Graspy being displayed as well?

Reply:
like, historically, how many times graspy was downloaded prior to the merge? i dont care about that. it was only doing downloads/week anyway.